src/generator/AutoRest.Go/TemplateModels/MethodTemplateModel.cs

```diff
@@ -405,5 +389,10 @@ public string AutorestError(string phase, string response = null, string paramet
                                  : string.Format("autorest.NewErrorWithError(err, \"{0}.{1}\", \"{2}\", {3}, \"{4}\")", PackageName, Owner, ScopedName, response, phase);
 
         }
+
+        public string ValidationError()
+        {
+            return $"validation.NewErrorWithValidationError(err, \"{PackageName}.{Owner}\",\"{ScopedName}\")";
```

Comment:
cool!

---
In reply to: [77408342](https://github.com/Azure/autorest/pull/1397#discussion_r77408342) [](ancestors = 77408342)